projects/packages/publicize/src/class-rest-controller.php

```diff
@@ -204,6 +240,27 @@ public static function get_social_product_info() {
 		);
 	}
 
+	/**
+	 * Dismisses a notice to prevent it from appearing again.
+	 *
+	 * @param WP_Request $request The request object, which includes the parameters.
+	 * @return WP_REST_Response|WP_Error True if the request was successful, or a WP_Error otherwise.
+	 */
+	public function update_dismissed_notices( $request ) {
+		$notice            = $request->get_param( 'notice' );
```

Comment:
We can safely assume that `$notice` is set to a valid value (only `instagram` right now), because the schema doesn't allow for anything else.